lib/tasks/styleguide.rake

```diff
@@ -0,0 +1,13 @@
+namespace :styleguide do
+  desc "Generate Live Styleguide"
+  task :generate => :environment  do
+    system "rspec spec/features/populate_styleguide.rb"
+    system "ruby testing.rb"
+  end
+
```

Comment:
Should we have a guard clause ensuring the user has the app installed?  And provide an informative error message if not? Or is that a given?